felix/bpf-gpl/nat_lookup.h

```diff
@@ -152,8 +152,9 @@ static CALI_BPF_INLINE struct calico_nat_dest* calico_nat_lookup(ipv46_addr_t *i
 		goto skip_affinity;
 	}
 
+	ipv46_addr_t dst = *ip_dst;
```

Comment:
This is needed to make the verifier happy when loading connect_balancer_debug_co-re_v4.o

Reply:
Ohhh interesting, so we can finally do ctlb core! :tada: